docs/man/man1/kubectl.1

```diff
@@ -16,7 +16,7 @@ kubectl \- kubectl controls the Kubernetes cluster manager
 kubectl controls the Kubernetes cluster manager.
 
 .PP
-Find more information at 
```

Comment:
`hack/verify-gendocs.sh` is complaining about this change. If it was intended, it needs to be made in the command this output is generated from [here](https://github.com/endocode/kubernetes/blob/kbeecher/kubectl_diff/pkg/kubectl/cmd/cmd.go#L117).